src/zmalloc.c

```diff
@@ -89,9 +89,21 @@ extern void jemk_free(void* ptr);
 #define realloc_pmem(ptr,size) memkind_realloc(MEMKIND_DAX_KMEM,ptr,size)
 #define free_dram(ptr) jemk_free(ptr)
 #define free_pmem(ptr) memkind_free(MEMKIND_DAX_KMEM,ptr)
+
+static void* kmem_dax_memory_variant = 0;
 #endif
 
 #ifndef USE_MEMKIND
+
+void zmalloc_kmem_dax_default_mode(void) {
```

Comment:
Why not to use single function with different arguments?

Reply:
Thanks for looking at this :) This is similar approach to existing zmalloc_set_pmem_mode(void). I tried to use int casted "KMEM DAX variants" defined in server.h but this file isn't included in zmalloc.c and I didn't want to redefine them in second place. So I stayed with set_*_mode()